app/wasm/testdata/version.txt

```diff
@@ -0,0 +1 @@
+v0.4.0
```

Comment:
Is this the contract version? A bit confused what this is haha

Reply:
It is the version of the contract we use, so you can match it to which bindings: https://github.com/confio/osmosis-bindings/tree/v0.4.0/packages/bindings

We changed these names a few times, and use this to make sure the Go and Rust stay in sync. Once this is ready to go in a tagged release, we will make a 1.0 of osmo-bindings and this will be stable. But for now, it lets us track renames more easily